truss/templates/server/common/logging.py

```diff
@@ -13,6 +13,15 @@
 )
 
 
+class HealthCheckFilter(logging.Filter):
+    def filter(self, record: logging.LogRecord) -> bool:
+        # for any health check endpoints, lets skip logging
+        return (
+            record.getMessage().find("/") == -1
```

Comment:
won't this filter out most of the logs as most of them contain a `/` ?

Reply:
You're right, I misread the API here. Fixed when i tested and all works in recent version.